package.json

```diff
@@ -12,29 +12,29 @@
     "publish-prod": "npm publish publish/"
   },
   "dependencies": {
-    "@angular/animations": "6.0.3",
-    "@angular/common": "6.0.3",
-    "@angular/compiler": "6.0.3",
-    "@angular/core": "6.0.3",
-    "@angular/forms": "6.0.3",
-    "@angular/http": "6.0.3",
-    "@angular/platform-browser": "6.0.3",
-    "@angular/platform-browser-dynamic": "6.0.3",
-    "@angular/router": "6.0.3",
+    "@angular/animations": "6.0.4",
+    "@angular/common": "6.0.4",
+    "@angular/compiler": "6.0.4",
+    "@angular/core": "6.0.4",
+    "@angular/forms": "6.0.4",
+    "@angular/http": "6.0.4",
+    "@angular/platform-browser": "6.0.4",
+    "@angular/platform-browser-dynamic": "6.0.4",
+    "@angular/router": "6.0.4",
     "@types/lscache": "^1.0.7",
     "classlist.js": "^1.1.20150312",
     "core-js": "^2.4.1",
-    "ids-enterprise": "^4.8.0-dev.20180528",
+    "ids-enterprise": "^4.7.0-rc.2",
```

Comment:
https://www.npmjs.com/package/ids-enterprise/v/4.8.0-dev.20180611 is newer... Or 4.7.0 (not RC)

Reply:
... as above.  ;-)